docs/modules/operation/pages/deep-dive/flows/sentinel/message-broker/kafka.adoc

```diff
@@ -25,47 +37,36 @@ ssh -p 8301 admin@localhost
 config:edit org.opennms.sentinel.controller
 config:property-set location SENTINEL<1>
 config:property-set id 00000000-0000-0000-0000-000000ddba11<2>
-config:property-set http-url http://core-instance-ip:8980/opennms<3>
 config:update
 ----
-
-<1> A location string is used to assign the Sentinel to a monitoring location in your environment that has Minions.
-<2> Unique identifier used as a node label for monitoring the Sentinel instance within {page-component-title}.
-<3> URL that points to your core {page-component-title} instance.
+<1> A location string is used to assign the Sentinel to a monitoring location. This can be an existing location or a new location and does not impact the messages that this Sentinel will process.
```

Comment:
```suggestion
<1> A location string is used to assign the Sentinel to a monitoring location.
This can be an existing location or a new location and does not impact the messages that this Sentinel will process.
```